<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Receipt_Entry</class>
 <widget class="QWidget" name="Receipt_Entry">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* =========================
   MAIN WINDOW
   ========================= */
QWidget {
    background-color: #191919;
    color: white;
    font-family: Segoe UI;
    font-size: 13px;
}

/* =========================
   LOGIN CARD (ORANGE BORDER)
   ========================= */
QFrame {
    background-color: transparent;
    padding: 16px;
}

/* =========================
   TEXT
   ========================= */
QLabel {
    color: #E0E0E0;
}

QLabel#lblTitle {
    color: #FF9800;
    font-size: 24px;
    font-weight: bold;
}

/* =========================
   INPUT FIELDS (BORDER STYLE)
   ========================= */
QLineEdit {
    background-color: transparent;
    color: white;
    border: 1.5px solid #FF9800;
    border-radius: 8px;
    padding: 4px;
}

QComboBox {
    background-color: transparent;
    color: white;
    border: 1.5px solid #FF9800;
    border-radius: 8px;
    padding: 4px;
}

QDateEdit {
    background-color: transparent;
    color: white;
    border: 1.5px solid #FF9800;
    border-radius: 8px;
    padding: 4px;
}

QLineEdit:focus {
    border: 2px solid #FFA726;
}

/* =========================
   LOGIN BUTTON (OUTLINE STYLE)
   ========================= */
QPushButton {
    background-color: transparent;
    color: #FF9800;
    font-weight: bold;
    border: 2px solid #FF9800;
    border-radius: 10px;
    padding: 10px;
}

QPushButton:hover {
    background-color: rgba(255, 152, 0, 25);
}

QPushButton:pressed {
    background-color: rgba(255, 152, 0, 60);
    color: black;
}

/* =========================
   DISABLED BUTTON
   ========================= */
QPushButton:disabled {
    border-color: #555;
    color: #555;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="FrameInfo">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLineEdit" name="leInvoiceNumber">
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Invoice No.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>100</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Explanation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leExpense">
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Expense</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="displayFormat">
           <string>yyyy/M/dd</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbExpenseCenter">
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="maxVisibleItems">
           <number>3</number>
          </property>
          <property name="placeholderText">
           <string>Expense Center</string>
          </property>
          <property name="modelColumn">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbExpenseType_2">
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="maxVisibleItems">
           <number>3</number>
          </property>
          <property name="placeholderText">
           <string>Expense Type</string>
          </property>
          <property name="modelColumn">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbCompany">
          <property name="maximumSize">
           <size>
            <width>700</width>
            <height>734</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>S</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>T</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lblSelectPicture">
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>300</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="text">
              <string>No Picture Selected</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnBrowse">
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>90</height>
              </size>
             </property>
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnClear">
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>90</height>
              </size>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
